### DISCUSSION TITLE
✅ 6 篇 Discussion ➕ 6 篇 Comments 拿滿 70 點的心得與小技巧分享

### DISCUSSION THREAD

**Discussion by xxrjun** [1101-GS4719-C/Python-Community-GS4719-C#1302] [General]
![image](https://user-images.githubusercontent.com/40348319/235138949-0b7dde90-203f-408c-a48f-eaec5f863fa8.png)

> 上週意外得到老師賞識，老師幾根手指頭直接讓我分數爆滿 4x ➡️ 7x (超級感謝老師 :star_struck:)
> 終於拿滿這個分數，過程中也累積了一些心得，所以以此篇做個紀錄~

內容大概會分成

- [如何獲取小 AI 賞識 —— 先以『_我_』為主詞](#如何獲取小-ai-賞識--先以我為主詞)
- [在 GitHub 寫文章必備的 Markdown 語法 —— 更好的寫作格式](#在-github-寫文章必備的-markdown-語法--更好的寫作格式)
- [其他小技巧](#其他小技巧)
- [貼文回顧](#貼文回顧)

## 如何獲取小 AI 賞識 —— 先以『_我_』為主詞

> ⚠️ 這個部分僅為個人觀察，不一定正確

我目前觀察下來最主要有兩點

- 文字篇幅至少要 _20_ 字左右
- 以 **"我"** 為主詞的句子需要佔整篇文章一定**比例**

其中比例的部分蠻關鍵的，因此主要為分享知識的文章需要在「知識介紹」與「個人觀點」上做一些平衡。

另外再詳細一點的話如下

- 多用 **"我"** 這個主詞，權重最高
- 後面接 **"認為"** 、 **"覺得"** 、 **"想"** 等等表達個人想法意見，權重次之
- 後面接其他動詞也可以，不過權重應該比第二項低

## 在 GitHub 撰寫文字必備的 Markdown 語法 —— 更好的寫作格式

在 GitHub 上最常見撰寫文字的方式有兩種，一種是使用 HTML，另一種就是使用 Markdown 語法。

> 我個人比較推薦使用 Markdown 語法，因為比較簡單。
> 不過如果需要更多樣化的排版，像是圖片大小、文字至中等等效果，我就會使用 HTML

> 有在使用 Notion 或是 HackMD 的同學想必對這部分非常熟悉 👍

基本上只要掌握幾項重要的語法就可以增加文章的可讀性，以下是我覺得最重要的幾項

- 標題 (Heading1 ~ Heading6)
- 清單
- 文字粗體、斜體
- 連結、圖片
- 程式碼

### 標題 (Heading1 ~ Heading6)

```markdown
# Heading1

## Heading2

...

###### Heading6
```

其中 H1 跟 H2 標題下方會有一條線，其他的則沒有~

### 清單

有序清單

```markdown
1. 有序清單
2. 有序清單
3. 有序清單
```

無序清單

```markdown
- 無序清單
- 無序清單
- 無序清單
```

### 文字粗體、斜體

```markdown
**粗體**  # 將要粗體的文字選取起來按下 ctrl + b 即可
_斜體_     # 將要粗體的文字選取起來按下 ctrl + i 即可
```

### 連結、圖片

連結 [This is Google](https://www.google.com) 寫成 Markdown 語法如下

```markdown
[This is Google](https://www.google.com)
```

要將圖片插入 Markdown 語法如下

```markdown
![Google 圖片](https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png)
```

在 GitHub 上面插入圖片通常會使用網址，現在可以直接拖拉圖片到 GitHub 的編輯器上面，GitHub 就會自動幫你上傳圖片到專案裡面，並且產生網址，真的非常方便!

### 程式碼

使用一對 ` 符號包起來，這個符號在鍵盤上通常在 tab 鍵的上面

```markdown
`單行程式碼`
```

上面範例的效果會是 `單行程式碼`

如果要插入多行程式碼的話，則需要使用一對三個 ` 符號包起來。  
並且可以在第一個 ``` 符號後面加上程式語言，這樣 GitHub 就會幫你加上程式碼的 Hightlight。

````markdown
```python
# 多行程式碼
print("Hello World")
print("Hello World")
```
````

效果如下

```python
# 多行程式碼
print("Hello World")
print("Hello World")
```

如果想要更深入了解的話可以參考以下文件 ⬇️

- [GitHub Markdown —— Basic writing and formatting syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

### 使用 emoji

使用 emoji 可以增加文章可讀性，也可以讓文章更加生動。這個部分我推薦 [ikatyang/emoji-cheat-sheet](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md) 這個專案，裡面有非常多 emoji 可以使用，而且還有分類，並同時包含 emoji 的 markdown 語法。

### 看一些科技時事文章，增加靈感

我自己平常蠻愛看一些科技時事的文章，像是 [Inside](https://inside.com/) 這個網站，或是 [Hacker News](https://news.ycombinator.com/) 這個網站，這些網站都會有一些科技時事的文章，有時候看到一些文章會有一些想法與靈感，於是一篇文章就這樣誕生了。

## 貼文回顧

> 每篇文章我大概都需要花 40 ~ 60 分鐘完成，因此能夠獲得老師的賞識真的非常開心
>
> 我自己覺得 6 篇 Dicussion + 6 篇 Comment 能夠拿滿 70 點應該也算是蠻有效率的(?

- #1160 
- #1164 
- #1199 
- #1203 
- #1171 
- #1270 

### Summary

沒意外的話這應該是期末考結束前最後一篇了
之後如果有機會可能會分享一下一學期修 :two::nine: 分的生活長怎樣
有資工演算法、資管演算法、資管專題...

最後再次感謝老師 @benctw

**Comment by brian431** [comment]
謝謝分享，個人之前修課時也是靠著經常發文達到七十分。不過這學期更改戰術了，變成看到能留言的就留一下。慢慢累積起來總會到七十分的，希望大家都能多多發一些好留言的文。

**Comment by YoYooHuang** [comment]
這實在太有用了，這拯救了有分數危機的我🥲。之後我會多加嘗試你裡面提到的技巧，謝謝分享！
